Dockerfiles/midstream/run_tests.py

```diff
@@ -4,45 +4,71 @@
 import json
 import unittest
 import subprocess
+import logging
 from os import path
 
 
+TEST_WORKING_DIR= os.getcwd()
+# global variables set
+OPERATOR_DIR=TEST_WORKING_DIR+"/operator_dir/"
+OPERATOR_WORK_DIR=TEST_WORKING_DIR+"/test_operator_work_dir/"
+WORK_DIR=TEST_WORKING_DIR+"/output/"
+PLAYBOOKS_DIR="/operator-test-playbooks/"
+ERROR_MESSAGE_PATH=TEST_WORKING_DIR+"/.errormessage"
```

Comment:
What will happen if somebody set wrong current working directory ?
e.g. you  do not have a permissions to write there.

Reply:
If the working directory is set wrong. It should fail.